web/client/utils/__tests__/FilterUtils-test.js

```diff
@@ -138,7 +139,7 @@ describe('FilterUtils', () => {
 
         let filter = FilterUtils.toCQLFilter(filterObj);
         expect(filter).toExist();
-        expect(filter).toBe("(\"attribute1\"='value1') AND (INTERSECTS(the_geom, Polygon((1 2, 2 3, 3 4, 4 5, 5 6, 1 2))))");
+        expect(filter).toBe("(\"attribute1\"='value1') AND (INTERSECTS(the_geom,SRID=4326;Polygon((1 2, 2 3, 3 4, 4 5, 5 6, 1 2))))");
```

Comment:
Please test also when geometry projection is missing (generating a CQL_FILTER without a specific SRID

Reply:
Done